bazel/external/libprotobuf_mutator.BUILD

```diff
@@ -0,0 +1,27 @@
+cc_library(
```

Comment:
@jmillikin-stripe I'm not super happy with how this turned out, I don't like non-trivial BUILD files in Envoy, but this was simplest vs. getting protobuf as a Bazel dep and the cmake build system to play nice. I've filed https://github.com/google/libprotobuf-mutator/issues/91 to request that we get proper upstream Bazel support for this project. I think the current solution is fine as this is a test-only dependency and it's a pretty simple library, we don't care about tests in this case.

Reply:
You might be able to simplify this with globs:
```
srcs = glob(["**/*.c", "**/*.h"], exclude=["**/*_test.cc"])
```

Reply:
@jmillikin-stripe can you take another look at what I've done for OS X? I'm not very happy with it, I would like something more like https://github.com/bazelbuild/bazel/issues/3510.